tests/testthat/test-check-schema-df.R

```diff
@@ -17,6 +18,7 @@ test_that("check_schema_df passes if all rows valid", {
 })
 
 test_that("check_schema_df fails if all rows invalid", {
+  skip_if_not(requireNamespace("jsonvalidate", quietly = TRUE))
```

Comment:
I realized after we talked that there's already a `skip_if_not_installed()` function, which would be shorter, but this works too

Reply:
Haha, I should have noticed that when I was doing it, too, especially with auto-complete. I like `skip_if_not_installed()` better since it is more clear. I might change to that. It shouldn't negatively affect the OS tests.